packages/odf/modals/configure-mons/configure-mons.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import {
+  StorageClusterModel,
+  getName,
+  getNamespace,
+  useCustomTranslation,
+} from '@odf/shared';
+import { LoadingInline } from '@odf/shared/generic/Loading';
+import { ModalBody, ModalFooter, ModalHeader } from '@odf/shared/modals/Modal';
+import { k8sPatch } from '@openshift-console/dynamic-plugin-sdk';
+import { ModalComponent } from '@openshift-console/dynamic-plugin-sdk/lib/app/modal-support/ModalProvider';
+import { Trans } from 'react-i18next';
+import {
+  Alert,
+  Button,
+  Flex,
+  FlexItem,
+  Modal,
+  ModalVariant,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import { useSafeK8sGet } from '../../hooks';
+import { useODFNamespaceSelector } from '../../redux';
+
+const patch = [
+  {
+    op: 'replace',
+    path: '/spec/managedResources/cephCluster/monCount',
+    value: 5,
+  },
+];
+
+const LowMonAlertModal: ModalComponent = ({ closeModal }) => {
+  const { odfNamespace } = useODFNamespaceSelector();
+  const { t } = useCustomTranslation();
+
+  const [errorMessage, setErrorMessage] = React.useState('');
+  const [inProgress, setProgress] = React.useState(false);
+
+  //Todo(bipuladh): Update it to use information from the alert
```

Comment:
sorry for adding such a strange suggestion on the PR, but I am adding comments prefixed with term `// ToDo (epic 4422)` so that it will be easier to search for all such instances later and make updates related to multiple storagecluster epic...
```suggestion
  // ToDo (epic 4422) (bipuladh): Update it to use information from the alert
```